benches/benches/vm_set/blockchain.rs

```diff
@@ -75,6 +75,7 @@ impl BenchDb {
             tmp_dir.path(),
             None,
             Default::default(),
+            -1,
```

Comment:
There are a few places that we're still using `-1`. Is that okay?

Reply:
I have let in benches because benches didn't used the test value before and I don't want them to be slowed down by a new limit (maybe it don't change anything but there is no risk for benches I think)